### ISSUE TITLE
[A11y]:VA Forms System Core [Demo Content]: Magnification - CONTINUE button is partially cut off when viewing at 300% zoom

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#45723]
### Point of contact

Angela Fowler

### Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

### Details

...
## User Story or Problem Statement

As a user who magnifies content on a page, I expect all content to be visible without horizontal scroll.

## Details

Currently, the CONTINUE button is partially cut off from view at 300% magnification (1280px screen size).
## Acceptance Criteria

- [ ] CONTINUE button on all form steps is completely visible and not cut off

## Environment

* macOS Monterey, `<Version 12.4>`

* Chrome, `<Version 102.0.5005.61>`

* VoiceOver

* Staging
* 

### Reproduction steps

1.
2.
3.
...
1. Enter `https://staging.va.gov/burial-poc-v6/` in browser address bar

2. Navigate through all of the form steps with screen set to 1280px and zoom to 300%

3. Verify that the CONTINUE button is partially cut off

### Proposed solution or next steps

...

### References, articles, or WCAG support

1.
2.
3.
...

### Type of issue

- [ ] Screenreader
- [ ] Keyboard
- [ ] Focus
- [ ] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [ ] Something else
